### PR TITLE
docs(README): Fix several broken links

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ integrations. It consists of the following tools which can be combined into a *h
   gathered in preceding stages and returns a list of policy violations, e.g. to flag license findings.
 * [*Reporter*](#reporter) - presents results in various formats such as visual reports, Open Source notices or
   Bill-Of-Materials (BOMs) to easily identify dependencies, licenses, copyrights or policy rule violations.
-* [*Notifier*](./notifier) - sends result notifications via different channels (like
-  [emails](./examples/example.notifications.kts) and / or JIRA tickets).
+* *Notifier* - sends result notifications via different channels (like [emails](./examples/example.notifications.kts)
```
Comment (sschuberth) [inline README.md:63]: Commit message:

> The notifier is not a separate tool

Actually it is, both as a CLI command and as a Gradle module, so the commit message should be corrected in this regard.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Comment by sschuberth** [comment]
Please also rebase to skip the first commit, which should be equal to @nnobelis's change.

File: README.md [hunk 2]
```diff
@@ -46,18 +46,20 @@ your own copy as nothing on the internet is forever
...
-  Version Control System (VCS) or other means are used to retrieve the source code.
-* [*Scanner*](#scanner) - uses configured source code scanners to detect license / copyright findings, abstracting
-  the type of scanner.
-* [*Advisor*](#advisor) - retrieves security advisories for used dependencies from configured vulnerability data
-  services.
-* [*Evaluator*](#evaluator) - evaluates custom policy rules along with custom license classifications against the data
-  gathered in preceding stages and returns a list of policy violations, e.g. to flag license findings.
-* [*Reporter*](#reporter) - presents results in various formats such as visual reports, Open Source notices or
-  Bill-Of-Materials (BOMs) to easily identify dependencies, licenses, copyrights or policy rule violations.
+* [*Analyzer*](./website/docs/tools/analyzer.md) - determines the dependencies of projects and their metadata,
```
[the remark this answers is not in the document]
Comment (fviernau) [reply README.md:49]: Shall I close this PR then ?

[the remark this answers is not in the document]
Comment (sschuberth) [reply README.md:49]: I don't think we should close this completely, as there are some valid changes in here that would fix the link check errors I'm running into e.g. [here](https://github.com/oss-review-toolkit/ort/actions/runs/6817585531/job/18541391376#step:4:42).

So basically, I believe we should drop the changes in *this* hunk, but keep the 3 link fixes below.

**Review by sschuberth — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -429,7 +429,7 @@ or [packages](./plugins/package-managers/spdx/src/funTest/assets/projects/synthe
 
 Taking an ORT result file with an *analyzer* result as the input (`-i`), the *downloader* retrieves the source code of
 all contained packages to the specified output directory (`-o`). The *downloader* takes care of things like normalizing
-URLs and using the [appropriate VCS tool](./downloader/src/main/kotlin/vcs) to check out source code from version
+URLs and using the [appropriate VCS tool](./plugins/version-control-systems) to check out source code from version
```
Comment (sschuberth) [inline README.md:432]: Thanks!

**Review by sschuberth — APPROVED** [review]

**Comment by fviernau** [comment]
Failing tests we're all unrelated.

**Pull request merged**